New faculty in ML at Georgia Tech (both primary and non CS)

animesh-garg opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

animesh-garg committed:
Recording Faculty Moves to Georgia Tech (Sp 2023)

- Animesh Garg: University of Toronto -> Georgia Institute of Technology
    - https://research.gatech.edu/animesh-garg

- Alexander Travis Adams, New faculty at Georgia Institute of Technology
    - https://www.cc.gatech.edu/people/alexander-t-adams
    - https://dblp.org/pid/128/9421.html

animesh-garg committed:
Recording Faculty Moves: Diyi Yang: GT -> Stanford

Diyi Yang: Georgia Institute of Technology -> Stanford Univeristy
    - https://cs.stanford.edu/~diyiy/

animesh-garg committed:
Adding New Faculty at GT: Kexin Rong

Kexin Rong, Georgia Tech, Started in Fall 2022
- https://dblp.org/pid/192/4560.html
- https://scs.gatech.edu/people/kexin-rong

animesh-garg committed:
Removing Faculty with >25% appointment in Industry

Removing Faculty with >25% appointment
- Sanja Fidler 50-50: University of Toronto - Nvidia
- Daniel Wigdor 50-50: University of Toronto - Meta
- Alec Jacobsen 50-50: University of Toronto - Adobe

This is in line with exclusion of similar others such as - Dieter Fox: 50-50: University of Washington - Nvidia
- Anima Anandkumar 50-50: CalTech - Nvidia

animesh-garg committed:
Adding CS at GT Faculty

- Sehoon Ha

animesh-garg committed:
ML Faculty at GT

ML Faculty at Georgia Tech.
These faculty members can solve supervise students in the ML program. ML is not a department, and students apply through CS department.

Hence a CS student can be sole supervised by these faculty members.

animesh-garg committed:
New faculty at GT from 2020-2022

animesh-garg commented:
**Note 1**. Georgia Tech College of Computing operates at the level of College of Engineering with [5 schools](https://www.cc.gatech.edu/schools) and several [specialized PhD programs](https://www.cc.gatech.edu/degree-programs) including the CS. This is similar to [SCS at CMU](https://www.cs.cmu.edu/academics/overview-programs) with multiple departments, but markedly different from most other schools where CS is a *single* department or often joint with EE.

**Primary CS Faculty** 
1. Alexander Travis Adams
2. Christopher J. MacLellan
3. Jan van den Brand
4. Jennifer G. Kim
5. Judith Uchidiuno
6. Kexin Rong
7. Sehoon Ha
8. Vernelle A. A. Noel

**Note 2**: Georgia Tech ML program allows all program faculty to supervise solely any ML students. ML students applying through CS program can pick any of the program faculty as primary (and sole) supervisors. 
[Link 1](https://ml.gatech.edu/people/faculty/phdprogramfaculty), [Link FAQ](https://ml.gatech.edu/content/program-faq)
This enables the following faculty members to work with a College of Computing (aka CS) student. 

**Non CS faculty with sole supervision rights to students in CS**
1. Arkadi Nemirovski
2. Ashwin Pananjady
5. Chethan Pandarinath
6. Christopher J. Rozell
9. Chuanyi Ji
10. Enlu Zhou
11. Eva L. Dyer
12. Evangelos A. Theodorou
13. Ghassan Al-Regib
14. Guanghui Lan
15. Juba Ziani
16. Justin K. Romberg
17. Larry P. Heck
18. Madhavan Swaminathan
19. Mark A. Davenport
20. Molei Tao
21. Nikolaos V. Sahinidis
22. Panagiotis Tsiotras
23. Pascal Van Hentenryck
24. Patricio A. Vela
25. Peng Chen 0024
26. Saibal Mukhopadhyay
27. Shimeng Yu
28. Siva Theja Maguluri
29. Vidya Muthukumar
30. Wenjing Liao
31. Yao Xie
32. Yongxin Chen

**Change of Affiliation**
1. Animesh Garg (Toronto -> GT)
2. Diyi Yang (GT -> Stanford)
3. Pan Li (Purdue -> GT)
4. Sham Kakade (UW -> Harvard)

**Removals** (>25% industry appointment)
This is to maintain consistent application of policies. For instance the same rule excludes Dieter Fox and Anima Anandkumar.
1. Sanja Fidler
2. Daniel Wigdor
3. Alec Jacobsen
4. Frank Dalleart

animesh-garg committed:
Removing Frank Dalleart

Frank Dalleart is at a startup and on leave from Georgia Tech
https://dellaert.github.io/

animesh-garg committed:
Fix affiliation for Sham Kakade

Sham has moved to Harvard. 
He has three separate entries for his name.

animesh-garg commented:
@emeryberger Could you help with this PR.
it will indeed help a lot of new students in discovery relevant folks at GT for upcoming fall. 
Let me know if there are questions. 
Thanks

emeryberger committed:
Merge branch 'gh-pages' into gh-pages

emeryberger commented:
I just resolved a number of conflicts. Please move all removed faculty (with <75% appointments) to `old/industry.csv` rather than just deleting their entries. Thanks.

animesh-garg committed:
added the faculty in industry list

as per suggestion from Emery, adding the faculty in the industry list instead of removal

animesh-garg committed:
Added two new faculty at GT

Bo Zhu: Moving from Dartmouth to GT
Bo Dai: Started as ML faculty in Spring 2023

animesh-garg committed:
Merge branch 'gh-pages' into gh-pages

animesh-garg commented:
I have now changed three things
1. Added two new faculty members
2. Added the Faculty members to the industry list
3. Fixed the merge conflict

emeryberger committed:
Merge branch 'gh-pages' into gh-pages

emeryberger commented on csrankings-b.csv:
```diff
@@ -614,7 +615,7 @@ Bo Yang 0011,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/1772.htm,H2Oo_MkAA
 Bo Yuan 0006,SUSTech,https://faculty.sustech.edu.cn/yuanb,MAdS9ZsAAAAJ
 Bo Zhang 0010,Tsinghua University,http://www.tsinghua.edu.cn/publish/csen/4623/2010/20101226104412516277601/20101226104412516277601_.html,NOSCHOLARPAGE
-Bo Zhu 0002,Dartmouth College,https://web.cs.dartmouth.edu/people/bo-zhu,LM_HjvIAAAAJ
+Bo Zhu 0002,Georgia Institute of Technology,https://web.cs.dartmouth.edu/people/bo-zhu,LM_HjvIAAAAJ
```
Please add some evidence of this new affiliation as a comment.

dartmouth-physics-ai replied:
Hi Emery, I confirm this move. I will announce it on my website soon. Thank you!

-- Bo

toqitahamid commented on csrankings-k.csv:
```diff
@@ -577,6 +577,7 @@ Kevin Waugh,Open University UK,http://stem.open.ac.uk/people/kgw9,l5ryKEkAAAAJ
 Kevin Y. Yip,Chinese University of Hong Kong,http://www.cse.cuhk.edu.hk/~kevinyip,pAD5GlEAAAAJ
 Kevyn Collins-Thompson,University of Michigan,http://www-personal.umich.edu/~kevynct,q__pgY9AkUQJ
 Kewei Tu,ShanghaiTech University,https://sist.shanghaitech.edu.cn/sist_en/2020/0814/c7582a54884/page.htm,5gi3Pm0AAAAJ
+Kexin Rong,Georgia Institute of Technlogy,https://kexinrong.github.io/,C8LDYK0AAAAJ
```
There is a typo in the institution's name. It should be "Georgia Institute of **Technology**" instead of "Georgia Institute of **Technlogy**"

toqitahamid commented on csrankings-e.csv:
```diff
@@ -728,6 +730,7 @@ Evangelos P. Markatos,University of Crete,https://www.csd.uoc.gr/CSD/index.jsp?c
 Evangelos Papapetrou,University of Ioannina,https://cs.uoi.gr/~epap,R6Ldg5UAAAAJ
 Evangelos Pournaras,University of Leeds,https://eps.leeds.ac.uk/computing/staff/6446/dr-evangelos-pournaras,EffH1CEAAAAJ
 Evangelos Sakkopoulos,University of Piraeus,https://www.unipi.gr/unipi/en/sakkopul.html,gLzYZ9wAAAAJ
+Evangelos A. Theodorou,Georgia Instutute of Technology,https://ae.gatech.edu/people/evangelos-theodorou,dG9MV7oAAAAJ
```
There is a typo in the institution's name. It should be "Georgia **Institute** of Technology" instead of "Georgia **Instutute** of Technology"

toqitahamid commented on csrankings-m.csv:
```diff
@@ -714,6 +715,7 @@ Marivic S. Tangkeko,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/
 Mariya Zheleva,University at Albany - SUNY,https://www.albany.edu/ceas/mariya-zheleva.php,eEqTPcwAAAAJ
 Marjan Sirjani,Mälardalen University,https://marjansirjani.github.io/Marjan-Sirjani/,TKnSrTQAAAAJ
 Marjo Kauppinen,Aalto University,https://people.aalto.fi/new/marjo.kauppinen,Wo8x6EUAAAAJ
+Mark A. Davenport,Georgia Insitute of Technology, https://mdav.ece.gatech.edu/index.html,DpuKx_oAAAAJ
```
There is a typo in the institution's name. It should be "Georgia **Institute** of Technology" instead of "Georgia **Insitute** of Technology"

emeryberger replied:
Fixing manually.
